dotCMS/src/main/webapp/html/portlet/ext/contentlet/publishing/view_publish_tool.jsp

```diff
@@ -311,3 +349,31 @@
 
     </form>
 </div>
+
+<div dojoType="dijit.Dialog" autofocus="false" id="deleteBundleActions" title='<%= UtilMethods.escapeSingleQuotes(LanguageUtil.get(pageContext, "bundle.delete.title" )) %>'>
+    <table class="sTypeTable">
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Do not use an attribute called class.](https://app.codacy.com/app/dotCMS/core/pullRequest?prid=4502981)